inventory.yml

```diff
@@ -0,0 +1,49 @@
+images:
```

Comment:
I think I prefer having TravisCI generate this in a pretest and exclude it from git.